airbyte-integrations/connectors/source-stripe/source_stripe/schemas/setup_attempts.json

```diff
@@ -186,7 +186,268 @@
           "$ref": "payment_intent.json"
         },
         "payment_method": {
-          "$ref": "payment_method.json"
```

Comment:
why are we not using the referenced schema anymore?

Reply:
It contains the `customer` object pointing to be the `object` instead of `string` for unknown reasons to me.

Reply:
Please refer to the description links above to get more info about the structure of the payment_method object.